scenario.py

```diff
@@ -62,7 +62,10 @@ def __init__(self, params, experiment_path):
 
         self.contributivity_list = []
 
-        self.epoch_count = 40
+        if 'epoch_counts' in params.keys():
```

Comment:
Harmonize between `epoch_counts `and `epoch_count`. I think it should be `epoch_count `?